src/documents/tutorial-developerIntroduction/DeveloperIntroductionToInfusionFramework-ConcludingRemarks.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Developer Introduction to the Infusion Framework - Concluding Remarks
+layout: default
+category: Tutorials
+---
+
+We've come to the end of our introduction to the core concepts of the Infusion framework. There are many topics we haven't covered, but with these foundational concepts, we hope you go on to further explore Infusion.
+
+We'll summarize what we've done below, then suggest some next steps.
+
+<div class="infusion-docs-note"><strong>Note:</strong> If you have feedback about this tutorial, we'd be glad to hear it via our <a href="https://wiki.fluidproject.org/display/fluid/IRC+Channel">IRC channels</a>, <a href="https://wiki.fluidproject.org/display/fluid/Mailing+Lists">mailing lists</a> or a <a href="https://github.com/fluid-project/infusion-docs/issues">GitHub issue on the documentation repo</a></div>
+
```

Comment:
I vaguely recall I had some rationale based on a conversation with someone about asking to file documentation issues in the GitHub tracker, but I cannot for the life of me remember the details, or if this conversation actually occurred. I'll update it to point to the JIRA tracker instead.